#!/bin/sh /usr/share/dpatch/dpatch-run
## 13-module-mismatch.dpatch by Daniel Baumann <daniel@debian.org>
##
## DP: Adjusts failure message with Ubuntu specific solution.

@DPATCH@

Index: VirtualBox-2.2.0_OSE/src/VBox/VMM/VM.cpp
===================================================================
--- VirtualBox-2.2.0_OSE.orig/src/VBox/VMM/VM.cpp
+++ VirtualBox-2.2.0_OSE/src/VBox/VMM/VM.cpp
@@ -357,9 +357,9 @@ VMMR3DECL(int)   VMR3Create(uint32_t cCP
                     break;
                 case VERR_VERSION_MISMATCH:
                 case VERR_VM_DRIVER_VERSION_MISMATCH:
-                    pszError = N_("The VirtualBox support driver which is running is from a different "
-                                  "version of VirtualBox.  You can correct this by stopping all "
-                                  "running instances of VirtualBox and reinstalling the software.");
+                    pszError = N_("The version of the VirtualBox kernel modules and the version of "
+                                  "VirtualBox application are not matching. You can correct this by "
+                                  "updating virtualbox and module-virtualbox packages.");
                     break;
                 default:
                     pszError = N_("Unknown error initializing kernel driver");
